WordPress/src/main/java/org/wordpress/android/ui/comments/CommentsListFragment.java

```diff
@@ -481,26 +514,29 @@ protected void onPostExecute(CommentList comments) {
             if (comments == null && !getActivity().isFinishing() && mErrorType != ErrorType.NO_ERROR) {
                 switch (mErrorType) {
                     case UNAUTHORIZED:
-                        if (mEmptyView == null || mEmptyView.getVisibility() != View.VISIBLE) {
+                        if (mFilteredCommentsView.emptyViewIsVisible()) {
```

Comment:
The previous logic was to show the toast when the empty view was **not** visible. So should this be `if (!mFilteredCommentsView.emptyViewIsVisible())`?